### ISSUE TITLE
bug: ion-alert is partially hidden by the keyboard (iOS)

### ISSUE THREAD

**Issue by robingenz** [ionic-team/ionic-framework#19561]
# Bug Report

**Ionic version:**

[x] **4.x**

**Current behavior:**
iOS: `ion-alert` is partially hidden by the keyboard.
![885425f3-b8ee-4ebe-9045-916da4dcb20f](https://user-images.githubusercontent.com/13857929/66290620-0a061580-e8e0-11e9-8936-c571e041b228.jpg)

**Expected behavior:**
iOS: `ion-alert` should be centered vertically after opening the keyboard (like in Android).

On Android it looks like this:
![e22125f3-b8ee-4c2a-b1c7-e16da4eda1c0](https://user-images.githubusercontent.com/13857929/66290680-26a24d80-e8e0-11e9-906c-c9b85a2bd7da.png)

**Steps to reproduce:**

1. Add `ion-alert` with inputs 
2. Run the app on iOS and open the keyboard by focusing on the input field of the `ion-alert`.

**Other information:**

**Ionic info:**

```
$ ionic info
Ionic:

   Ionic CLI                     : 5.4.1
   Ionic Framework               : @ionic/angular 4.10.0
   @angular-devkit/build-angular : 0.803.6
   @angular-devkit/schematics    : 8.3.6
   @angular/cli                  : 8.3.6
   @ionic/angular-toolkit        : 2.0.0

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : ios 4.5.4
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 4.1.1, (and 22 other plugins)

Utility:

   cordova-res : 0.6.0 
   native-run  : 0.2.8 

System:

   ios-deploy : 1.9.4
   NodeJS     : v10.15.3 (/usr/local/bin/node)
   npm        : 6.4.1
   OS         : macOS Mojave
   Xcode      : Xcode 11.0 Build version 11A420a
```

**Comment by robingenz** [comment]
Temporary workaround:
```
.sc-ion-alert-ios-h {
    position: absolute;
}
```

**Comment by somq** [comment]
Same bug on Android

**Comment by robingenz** [comment]
Fixed with #22425
